Adding highway shields in Peru

whubsch opened the pull request:
This PR adds the Peruvian national, departmental, and rural route shields to the map after some [tagging changes](https://community.openstreetmap.org/t/propuesta-de-cambio-en-las-relaciones-viales-en-peru/122393). Only the national routes required a new shield.

[Localhost link](http://localhost:1776/#map=13.06/-13.53598/-71.99538)
<img width="514" alt="Screenshot 2024-12-09 at 10 59 51 PM" src="https://github.com/user-attachments/assets/5203f75d-0dc5-43ed-ad3c-e752618b7d2e">

Pan-American, national, departmental, and rural highway shields in Peru
<img width="422" alt="Screenshot 2024-11-30 at 6 50 00 PM" src="https://github.com/user-attachments/assets/98ddaee8-2591-4fd6-97ef-419e97fbbdb6">

claysmalley commented on src/js/shield_defs.js:
```diff
@@ -3164,6 +3164,53 @@ export function loadShields() {
+        shields[`PE:departmental:${department}`],
+        shields[`PE:rural:${department}`],
+      ] = [
+        escutcheonDownShield(12, Color.shields.white, Color.shields.black),
```
These shields are visually closer to the state highway shields of Mexico (see line 584). Let's factor that out into a reusable shield definition and use it here.

Side note: this is making it really clear how much we need a draw function for badge shields, and maybe support for crossbars and accent colors in some of the functions...

whubsch replied:
Thanks, this should be addressed with the latest commit.

whubsch commented:
Ok, turning back to this after the holidays. I've moved the definition for the crossbar'd escutcheon shield so that it can be reused, and renamed those `svg` files to have a more generic name (`shield_escutcheon_crossbar_X`).

<img width="753" alt="Screenshot 2025-01-07 at 4 14 30 PM" src="https://github.com/user-attachments/assets/e17e3036-f2df-4cd2-82d2-bade5ffb7421" />

claysmalley reviewed (approved):
Thank you!
